tgui/packages/tgui/interfaces/Newscaster.js

```diff
@@ -0,0 +1,405 @@
+/**
+ * @file
+ * @author Original by ArcaneMusic (https://github.com/ArcaneMusic)
+ * @author Changes Shadowh4nD/jlsnow301
+ * @license MIT
+ */
+
+import { useBackend, useSharedState } from '../backend';
+import { BountyBoardContent } from './BountyBoard';
+import { UserDetails } from './Vending';
+import { BlockQuote, Box, Button, Divider, Modal, Section, Stack, Tabs, TextArea } from '../components';
+import { Window } from '../layouts';
+import { marked } from 'marked';
+import { sanitizeText } from "../sanitize";
+
+const CENSOR_MESSAGE = "This channel has been deemed as threatening to \
+the welfare of the station, and marked with a Nanotrasen D-Notice.";
+const TRUE = 1; // Look, fellas, I'm gonna be real with you
+const FALSE = 0; // This is just to clean up lots of === moments
+
+export const Newscaster = (props, context) => {
+  const { act, data } = useBackend(context);
+  const [screenmode, setScreenmode] = useSharedState(context, 'tab_main', 1);
+  return (
+    <Window
+      width={575}
+      height={560}>
+      <NewscasterChannelCreation />
+      <Window.Content scrollable>
+        <Stack fill vertical>
+          <Stack.Item>
+            <Tabs fluid textAlign="center">
+              <Tabs.Tab
+                color="Green"
+                selected={screenmode === 1}
+                onClick={() => setScreenmode(1)}>
+                Newscaster
+              </Tabs.Tab>
+              <Tabs.Tab
+                Color="Blue"
+                selected={screenmode === 2}
+                onClick={() => setScreenmode(2)}>
+                Bounty Board
+              </Tabs.Tab>
+            </Tabs>
+          </Stack.Item>
+          <Stack.Item grow>
+            {screenmode === 1 && (
+              <NewscasterContent />
+            )}
+            {screenmode === 2 && (
+              <BountyBoardContent />
+            )}
+          </Stack.Item>
+        </Stack>
+      </Window.Content>
+    </Window>
+  );
+};
+
+/** The modal menu that contains the prompts to making new channels. */
+const NewscasterChannelCreation = (props, context) => {
+  const { act, data } = useBackend(context);
+  const [publicmode, setPublicmode] = useSharedState(context, 'publicmode', 1);
+  const {
+    creating_channel,
+    viewing_channel,
+    name,
+    desc,
+  } = data;
+  if (!creating_channel) {
+    return null;
+  }
+  return (
+    <Modal
+      textAlign="center"
+      mr={1.5}>
+      <Stack vertical>
+        <Stack.Item vertical>
```

Comment:
```suggestion
        <Stack.Item>
```
I don't think that does anything here